frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -776,10 +778,10 @@ function ReaderDictionary:startSdcv(word, dict_names, fuzzy_search)
         -- We must ensure we will have some output to be readable (if no
         -- definition found, sdcv will output some message on stderr, and
         -- let stdout empty) by appending an "echo":
+
         cmd = cmd .. "; echo"
         local completed, results_str = Trapper:dismissablePopen(cmd, self.lookup_progress_msg or false)
         lookup_cancelled = not completed
-
```

Comment:
Please don't move around my valuable blank lines ! :)
(The comment above describe why `cmd = cmd .. "; echo"` - no need for a blank line to separate logical sections.)

Reply:
Done